docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -215,8 +213,11 @@ Step 2: Disk Formatting
 
        sgdisk     -n4:0:0        -t4:8309 $DISK
 
-   If you are creating a mirror or raidz topology, repeat the partitioning
-   commands for all the disks which will be part of the pool.
+   **Hints:**
+
+   - If you want to use only legacy or only UEFI booting, you do not have nessecary to do both partition, only legacy or only UEFI, respectively.
```

Comment:
We didn't have issues with obviousness of one-only partition for boot types, let's drop this hint.